procrastinate/app.py

```diff
@@ -157,10 +157,12 @@ def configure_task(self, name: str, **kwargs: Any) -> jobs.JobDeferrer:
 
         return tasks.configure_task(name=name, job_store=self.job_store, **kwargs)
 
-    def _worker(self, queues: Optional[Iterable[str]] = None) -> "worker.Worker":
+    def _worker(
+        self, queues: Optional[Iterable[str]] = None, name: Optional[str] = None
```

Comment:
I always wonder if optional parameters that are passed should be optional all the way of just at the first step. Any opinion on this ?

Reply:
No strong opinion, but yes on internal methods maybe it's not very useful (and a bit error prone) to have default values.